include/game/interactables_1/bouncy_bar.h

```diff
@@ -2,9 +2,9 @@
 #define GUARD_INTERACTABLE_BOUNCY_BAR_H
 
 #include "global.h"
-#include "game/interactable.h"
+#include "game/entity.h"
 
-void initSprite_Interactable_BouncyBar(Interactable *ia, u16 spriteRegionX,
```

Comment:
Are we gonna replace all `ia` with `me` eventually?

Reply:
If with 'eventually' you mean ASAP, then yes!

Though the category "interactable" should still exist, like it does for "enemy" - just not the type.